brayns/common/renderer/FrameBuffer.h

```diff
@@ -45,6 +44,7 @@ class FrameBuffer : public BaseObject
     virtual void resize(const Vector2ui& frameSize) = 0;
 
     virtual Vector2ui getSize() const { return _frameSize; }
+    const Vector2ui& getFrameSize() const { return _frameSize; }
```

Comment:
Is it needed to have getFrameSize() in addition to getSize()?

Reply:
Yes, as `getSize()` might be smaller if subsampling is used. Yet the 'real' frame buffer size is needed for the viewer.